apps/web/src/components/workflow/Preview/chat/ChatPreview.tsx

```diff
@@ -0,0 +1,99 @@
+import { Divider, useMantineColorScheme } from '@mantine/core';
+import { colors, Text } from '@novu/design-system';
+import { useAuthController } from '@novu/shared-web';
+import { useMutation } from '@tanstack/react-query';
+import { useEffect, useState } from 'react';
+import { useFormContext, useWatch } from 'react-hook-form';
+import { getLocalesFromContent } from '../../../../api/translations';
+import { IForm } from '../../../../pages/templates/components/formTypes';
+import { useStepFormErrors } from '../../../../pages/templates/hooks/useStepFormErrors';
+import { useStepFormPath } from '../../../../pages/templates/hooks/useStepFormPath';
+import { LocaleSelect } from '../common';
+import { ChatContent } from './ChatContent';
+import { ChatInput } from './ChatInput';
+import { previewChat } from '../../../../api/content-templates';
+import { errorMessage } from '../../../../utils/notifications';
+
+export function ChatPreview() {
```

Comment:
lets do a small refactoring in this file after Biswa in his PR applies the suggestions #5143

Reply:
I am merging this PR to my branch. I will handle this change with my refactor @LetItRock 
There are many overlapping changes in both PRs, Merging now will help in avoiding conflicts